CODEOWNERS

```diff
@@ -130,32 +130,29 @@
 /src/lib/staged_ledger/ @deepthiskumar @cmr
 /src/lib/staged_ledger_diff/ @deepthiskumar @cmr
 /src/lib/storage/ @bkase
-/src/lib/sync_handler/ @cmr
-/src/lib/sync_status/ @johnwu93
+/src/lib/sync_handler/ @cmr @wu-s-john
```

Comment:
I think we should keep @cmr the sole owner of this one at least until we do a deeper knowledge transfer. There's too much nuance that only @cmr is a aware of right now.

Adding yourself to `cli` is cool, though

Reply:
lol, but I wrote most of the functions in `sync_handler`

Reply:
I approve of this change

Reply:
ok, I guess I was mistaken